test/unit/middlewares/user-validator.test.js

```diff
@@ -308,6 +308,56 @@ describe("Middleware | Validators | User", function () {
     });
   });
 
+  describe("Test the update Self Validator for disabled_roles property", function () {
+    it("Allows the request to pass with disabled_roles property", async function () {
+      const req = {
+        body: {
+          disabled_roles: [],
+        },
```

Comment:
What happens if some other data is being sent with disabled_roles in backend?

Reply:
other data is being ignored as inside the conditional
```
if(req.body.disabled_roles){
......
}
```
we only giving `{disbaled_roles:req.body.disabled_roles}` in the updateFunction()

I have tested it even if we sent the other properties with it ignores those property, and if the properties are 
- username,
![image](https://github.com/user-attachments/assets/f74628f6-e350-4537-a56b-01f41240b766)

- roles
![image](https://github.com/user-attachments/assets/274863d6-3fae-4caf-afdd-09611755f16c)
not just the super_user role but all the roles are not allowed to be updated

which are handle in the above if block, it also return error of 400 series